presto-common/src/main/java/com/facebook/presto/common/RuntimeMetric.java

```diff
@@ -85,6 +90,13 @@ public String getName()
         return name;
     }
 
+    @JsonProperty
+    @ThriftField(2)
```

Comment:
Is there a specific reason to reuse an existing thrift id? 
When adding a new field in Thrift, we should use a new id. Otherwise, existing serialized Thrift struct cannot be parsed. This will also cause compatibility issue if the coordinator and the worker run different versions.